changes/process_changes.py

```diff
@@ -0,0 +1,124 @@
+# coding: utf-8
+
+from __future__ import absolute_import, division, print_function
+
+import argparse as A
+from funcy import memoize
+from glob import glob
+import os
+from os import path as P
+import re
+import yaml
+
+
+TN_RE = re.compile(r"[A-Z]\d\d\d-\d\d\d")
```

Comment:
This is unused: can you remove it? (it’s also wrong, as ticket numbers contain letters for Oct-Dec months)

Reply:
Good catch, thanks, I wonder why it wasn't caught by the style checker ...

Reply:
Because another module could do `process_changes.TN_RE`, so style checkers can’t detect this.